src/geff/metadata_schema.py

```diff
@@ -190,7 +190,7 @@ class GeffMetadata(BaseModel):
     )
 
     geff_version: str = Field(
-        ...,
+        default=version("geff"),
```

Comment:
this is a good pattern.  all calls to `Field()` with a positional default first argument should be replaced with `default=` (such as `sphere`, `track_node_props`, etc...), otherwise some type checkers interpret it as a required argument